tests/unit/other.tcl

```diff
@@ -574,6 +574,25 @@ start_server {tags {"other external:skip"}} {
     }
 }
 
+start_server {tags {"other external:skip"}} {
```

Comment:
Looks like this test caught something:
https://github.com/valkey-io/valkey/actions/runs/14957133612/job/42014310397?pr=2033#step:5:1403

```
io_threads.c:219 'id > 0 && id < server.io_threads_num' is not true
```

It looks like we are probably updating this value on the main thread while the thread is getting initialized on another thread. Maybe we should move these assertions to `createIOThread`. @uriyage Any other ideas?

Reply:
Yes, we can move it to the main-thread.

Reply:
Done.